lib/mix/lib/mix/tasks/test.ex

```diff
@@ -146,8 +146,8 @@ defmodule Mix.Tasks.Test do
 
   `CoverModule` can be any module that exports `start/2`, receiving the
   compilation path and the `test_coverage` options as arguments. It must
-  return an anonymous function of zero arity that will be run after the
-  test suite is done or `nil`.
+  return either `nil` or an anonymous function of zero arity that will be run after the
```

Comment:
sure. i just didn't want to modify the line bellow without need.
If you ask me, i would start the second sentence in a new line. 
"It must return either..."
this avoids unnecessary changes in lines in the future, as it give more room. WDYT?